_sass/comp/league.scss

```diff
@@ -13,4 +13,8 @@
       text-align: left;
     }
   }
+
+  .rookie-team {
+    font-style: italic;
```

Comment:
Suggestion: The booklet uses bold to denote rookie teams - perhaps we should follow here?

Reply:
Yeah, I looked at that. The booklet uses blue and bold. I didn't think that the blue was going to work as well here, though admittedly didn't explore bold.

Reply:
I agree blue would probably look odd. I'd be interested in seeing **bold**

Reply:
See description.

Reply:
I think italics works better. Bold is very obvious and draws attention from other teams...

Reply:
![image](https://user-images.githubusercontent.com/6527489/225270409-7cf435b2-8be7-4bb0-b2dd-a986e3e2146f.png)

The same is true for the styling in the booklet. I _think_ that's probably ok.